onepassword/onepassword.go

```diff
@@ -360,6 +397,22 @@ func (op *OnePassword) connectGet(vaultUUID, itemUUID string) map[string]string
 	}).(map[string]string)
 }
 
+//nolint:forcetypeassert
+func (op *OnePassword) connectGetFileContent(vaultQuery, itemQuery, fileUUID string) string {
+	return op.cache.Get(strings.Join([]string{vaultQuery, itemQuery}, "#"), func() any {
+		var ret string
+		if v, err := op.connect.GetFile(fileUUID, itemQuery, vaultQuery); err != nil {
+			op.l.Error("failed to retrieve file:", err.Error())
+			return ret
+		} else if c, err := op.connect.GetFileContent(v); err != nil {
+			op.l.Error("failed to retrieve file content:", err.Error())
+		} else {
+			ret = string(c)
+		}
+		return ret
```

Comment:
This is a really odd pattern.
I've talked to many people about it, and not a single one preferred the chained else statement.
There is really nothing like this in the standard library.